libs/gltfio/include/gltfio/AssetLoader.h

```diff
@@ -54,6 +54,9 @@ struct AssetConfiguration {
     //! specified, AssetLoader will use the singleton EntityManager associated with the current
     //! process.
     utils::EntityManager* entities = nullptr;
+
+    //! Optional default node name for anonymous nodes
+    char* defaultNodeName = "node";
```

Comment:
I would prefer the default to be `nullptr` because this would raise our memory footprint in large scene graphs.

Reply:
The default is "node" in original implementation and Im a little bit afraid if loadRenderable or loadLight need a non-null name

Reply:
For those 2 functions it only use name in error logging though